s3tests_boto3/functional/test_s3.py

```diff
@@ -12931,7 +13164,7 @@ def test_object_lock_put_obj_lock_invalid_years():
 @attr(operation='Test put object lock with invalid mode')
 @attr(assertion='fails')
 @attr('object-lock')
-def test_object_lock_put_obj_lock_invalid_years():
+def test_object_lock_put_obj_lock_invalid_mode():
```

Comment:
@mdw-at-linuxbox why was this test name changed?

Reply:
because the next test had the same name and this one was actually testing invalid years.